idalib/src/idb.rs

```diff
@@ -213,6 +214,36 @@ impl IDB {
         }
     }
 
+    pub fn get_cmt(&self, ea: Address, rptble: bool) -> String {
+        unsafe { idalib_get_cmt(ea.into(), rptble) }
+    }
+
+    pub fn set_cmt(
+        &self,
+        ea: Address,
+        comm: impl AsRef<str>,
+        rptble: bool,
+    ) -> Result<(), NulError> {
+        let s = CString::new(comm.as_ref())?;
+        unsafe {
+            set_cmt(ea.into(), s.as_ptr(), rptble);
+        }
+        Ok(())
+    }
+
+    pub fn append_cmt(
+        &self,
+        ea: Address,
+        comm: impl AsRef<str>,
+        rptble: bool,
+    ) -> Result<(), NulError> {
```

Comment:
I'd change this (and the other instance of `NulError`) to `IDAError` so it's easier downstream to handle errors from this library; should be possible by doing this:

```rust
let s = CString::new(...).map_err(IDAError::ffi)?;
```